tests/scripts/github-action-helper.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

This is more portable.

Reply:
done.